main/integration-tests/support/setup.js

```diff
@@ -131,6 +136,10 @@ class Setup {
       userRole: 'researcher',
       projectId,
     });
+
+    // Delay to prevent eventual consistency issues from affecting the next call.
+    await sleep(1000);
+
```

Comment:
Doesn't the 'create'  already introduce a delay?

Reply:
Thanks for catching this! I have updated the default deflake delay from 1 to 2 seconds. The 1 second delay is what the normal maximum time dynamodb needs to persist eventually consistent changes.